pkg/kubenest/controller/virtualcluster.node.controller/workflow/task/task.go

```diff
@@ -109,13 +113,17 @@ func NewDrainHostNodeTask() Task {
 	}
 }
 
+// nolint:dupl
```

Comment:
I'm a little unclear, why do we need such annotations? Is there public code that can be extracted?